src/Covenant/Internal/Rename.hs

```diff
@@ -126,7 +129,7 @@ data RenameError
     -- has @b@ undetermined.
     --
     -- @since 1.0.0
-    UndeterminedAbstraction
+    UndeterminedAbstraction (Vector (ValT AbstractTy)) (Vector (ValT Renamed))
```

Comment:
This needs a bumped `@since` I think, since it's a change not the Milestone 1 release I think.

Reply:
Not sure what version this should be? 1.1.1? 1.2.0? I'll go with 1.1.1, correct me if I'm wrong